Include/pystate.h

```diff
@@ -75,18 +75,39 @@ typedef struct {
     wchar_t *dll_path;      /* Windows DLL path */
 #endif
 
+    /* If greater than 0, enable isolated mode sys.path contains
+       neither the script's directory nor the user's site-packages directory.
+       Ignored if set to -1.
+
+       Related to Py_IsolatedFlag global configuration variable and -I command
+       line option. */
```

Comment:
Maybe "Initialized from Py_IsolatedFlag global configuration variable and set to 1 by -I command line option"?

Reply:
I updated the doc.